app/src/main/java/io/appium/uiautomator2/model/UiObjectElement.java

```diff
@@ -125,6 +126,13 @@ public String getAttribute(String attr) throws UiObjectNotFoundException {
                 result = AxNodeInfoHelper.getPackageName(toAxNodeInfo(element));
                 break;
             }
+            case HINT:
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
could it use a ternary operator?

Reply:
I thought so, but AS recommended this style, so just followed it to reduce warning by IDE at this time